20230323/mike.java

```diff
@@ -0,0 +1,58 @@
+/*
+Check if edges are enough, then DFS.
+*/
+class Solution {
+    Set<Integer> visited = new HashSet<>();
+    HashMap<Integer, LinkedList<Integer>> graph = new HashMap<>();
+
+    public int makeConnected(int n, int[][] connections) {
+        if((n-1) > connections.length) {
+            return -1;
+        }
+
+        for(int[] path : connections) {
+            int pcA = path [0];
```

Comment:
```suggestion
            int pcA = path[0];
```
what's wrong with the space? :trollface: